05_stringer/jessodri/main.go

```diff
@@ -0,0 +1,19 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+)
+
+var out io.Writer = os.Stdout
+
+type IPAddr [4]byte
+
+func (i IPAddr) String() string {
+	return fmt.Sprintf("%d.%d.%d.%d", i[0], i[1], i[2], i[3])
```

Comment:
Mine is exactly the same... I was kinda hoping that you could use , i... rather than i[0], i[1], i[2], i[3] but it seems you can't do that without have something like this...  
func (i IPAddr) String() string {
    tmp := make([]interface{}, len(i))
    for j, val := range ip {
        tmp[j] = val
    }
    return fmt.Sprintf("%d.%d.%d.%d", tmp...)
}
So I ended up leaving as I had it...